src/controllers/handlers/scene-stream-access-handlers/add-scene-stream-access-handler.ts

```diff
@@ -3,7 +3,7 @@ import { HandlerContextWithPath } from '../../../types'
 import { InvalidRequestError, StreamingAccessNotFoundError, UnauthorizedError } from '../../../types/errors'
 import { SceneStreamAccess } from '../../../types'
 import { PlaceAttributes } from '../../../types/places.type'
-const FOUR_DAYS = 4 * 24 * 60 * 60
+const FOUR_DAYS = 4 * 24 * 60 * 60 * 1000
```

Comment:
As this variable is used in multiple places, what do you think about abstracting this variable to consume it from a file?